### DISCUSSION TITLE
Unable to verify status: Please complete your profile name exactly as is in acadaemic affiliation documents

### DISCUSSION THREAD

**Discussion by radiantgurl** [community/community#125963] [GitHub Education]
I cannot. I dont want to reveal my full real name to the internet. Is there a way around this? I already have filled my billing information, why do i have to do this?

I dont want to have to doxx myself to get education benefits

**Comment by KlausJackson** [comment]
No, also your name on the billing information has to match your name on academic affiliation documents. 

The reason why you have to do this is very simple, this helps prevent fraud and ensures the program benefits are distributed appropriately to verified students, faculty, or researchers. They don't want individuals who are not students exploiting these benefits without contributing financially. Considerable effort has gone into developing these resources.

**Reply by radiantgurl** [reply]
> No, also your name on the billing information has to match your name on academic affiliation documents. 
> 
> The reason why you have to do this is very simple, this helps prevent fraud and ensures the program benefits are distributed appropriately to verified students, faculty, or researchers. They don't want individuals who are not students exploiting these benefits without contributing financially. Considerable effort has gone into developing these resources.

My billing name matches the one on my academic affiliation document

**Reply by KlausJackson** [reply]
Oh so you meant you don't want your username on github to be your real name, that's totally fine, GitHub doesn't force anyone to use their real name as their username, just upload your student ID and wait.

**Reply by radiantgurl** [reply]
It's automatically rejected by a bot.

**Reply by KlausJackson** [reply]
You can create a [ticket](https://support.github.com/tickets/personal/0) for github support and wait for their response about this, I have no other way.

**Reply by radiantgurl** [reply]
I have done that at the start of september, they haven't done anything about it.

**Comment by Galogly** [comment]
Do I have to change my @  user name with my full name too? because it doesn´t let me complete the process, and how do I change it?

**Reply by KlausJackson** [reply]
No, you don't have to change your username. But you want to change your username then you can go to account settings and change your username in the profile section.

**Reply by Galogly** [reply]
All rigth, thank you!
